third_party/s2n-bignum/include/s2n-bignum_aws-lc.h

```diff
@@ -19,40 +19,57 @@
 
 // Add modulo p_384, z := (x + y) mod p_384, assuming x and y reduced
 // Inputs x[6], y[6]; output z[6]
-extern void bignum_add_p384 (uint64_t z[static 6], const uint64_t x[static 6], const uint64_t y[static 6]);
+extern void bignum_add_p384(uint64_t z[static 6], const uint64_t x[static 6], const uint64_t y[static 6]);
```

Comment:
I think we should add to this file something about the meaning of "_alt" in the function names as in https://github.com/awslabs/s2n-bignum/blob/main/include/s2n-bignum.h#L27-L35

Reply:
good idea, added.